index.html

```diff
@@ -0,0 +1,87 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="style.css">
+  </head>
+
+  <body>
+    <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+
+    <!--
+
+    Hello Camper!
+
+    For now, the test suite only works in Chrome!
+    Please read the README below in the JS Editor before beginning.
+    Feel free to delete this message once you have read it.
+    Good luck and Happy Coding!
+
+    - The freeCodeCamp Team
+
+    -->
+    <script  src="js/index.js"></script>
+    <div class="container">
+      <div class="card">
+        
+          <h1 id="title">Survey Form</h1>
+      
+          <center><p id="description"><i>Please take a moment to fill our survey. Thank you</i></p></center>
+      
+          <fieldset>
+      <div class="form-group row">
+          <form id="survey-form">
+          <div class="form-group">
+            <label for="name" id="name-label">Name:</label><br>
+            <input class="form-control" id="name" type="text" placeholder="Enter name here" required></input>
+            </div>
+           
+      
+            <label for="email" id="email-label">Email:</label><br>
+            <input type="email" id="email" name="email" placeholder="email@gmail.com" required >
+            <br><br>
+      
+            <label for="name"id="number-label">Age:</label><br>
+            <input id="number" type="number"  placeholder="Age" min="18" max="50"></input>
```

Comment:
I suggest that you use semantic tags to lay out your HTML elements. It is a recommended practice in HTML/CSS due to compatibility with different devices. :smiley: